src/_P037_MQTTImport.ino

```diff
@@ -531,7 +609,7 @@ boolean Plugin_037(uint8_t function, struct EventStruct *event, String& string)
                   RuleEvent += event->String1;
                   RuleEvent += '=';
                   RuleEvent += unparsedPayload;
-                  eventQueue.addMove(std::move(RuleEvent));
+                  P037_addEventToQueue(event, RuleEvent);
```

Comment:
I think these kind of events need the eventvalues wrapped in quotes if they contain a comma or space.
Also they need to be wrapped in quotes not present in the payload and only if not already wrapped with quotes (there is a function for it)